lib/vdsm/virt/vmdevices/storagexml.py

```diff
@@ -130,6 +130,10 @@ def _update_source_params(params, disk_type, source):
         path = ''
     elif disk_type == 'block':
         path = source.attrib.get('dev')
+        reservations = vmxml.find_first(source, 'reservations', None)
+        if reservations is not None and \
```

Comment:
Don't use \ in vdsm code, use:

    if (reservations is not None and
            reservations.attrib.get('managed') == 'yes'):
        code...

Reply:
Ack, good to know. I have seen it used in some other place and assumed this is the preferred way.